lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -105,6 +95,14 @@ def call(env)
           # memory leaks.
           tracer.provider.context = Datadog::Context.new
         end
+
+        def resource_name_for(env, status)
+          if Datadog.configuration[:rack][:middleware_names]
+            return "#{env['RESPONSE_MIDDLEWARE']}##{env['REQUEST_METHOD']}"
```

Comment:
Minor, non-blocking thing: would it be cleaner to do a simple `if else` instead of a `return`? Usually I see `returns` only if you need to short-circuit a lot of code quickly (usually out of the general concern of `return`'s side effects.)

Reply:
Sure, I'll make that change 👍